locales/de.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Commands:": "Kommandos:",
+  "Options:": "Optionen:",
+  "Examples:": "Beispiele:",
+  "boolean": "boolean",
+  "count": "count",
+  "string": "string",
+  "array": "array",
+  "required": "erforderlich",
+  "default:": "Standard:",
+  "choices:": "Möglichkeiten:",
+  "generated-value": "Generierter-Wert",
```

Comment:
I notice these translated strings (`Standard:`, `Möglichkeiten:`, `Generierter-Wert`) start with a capital letter, even though they're typically not capitalized in other languages. Is this intentional?